flang/test/Lower/OpenMP/omp-parallel-allocate-clause.f90

```diff
@@ -0,0 +1,29 @@
+! This test checks lowering of OpenMP parallel Directive with
+! `PRIVATE` clause present.
+
+! RUN: %bbc -fopenmp -emit-fir %s -o  - | \
+! RUN:   FileCheck %s --check-prefix=FIRDialect
+
+!FIRDialect: func @_QPallocate_clause(%arg0: !fir.ref<f32>, %arg1: !fir.ref<f32>) {
+!FIRDialect-DAG: %[[A:.*]] = fir.alloca i32 {name = "{{.*}}Ea"}
+!FIRDialect-DAG: %[[B:.*]] = fir.alloca i32 {name = "{{.*}}Eb"}
+!FIRDialect-DAG: %[[X:.*]] = fir.alloca i32 {name = "{{.*}}Ex"}
+!FIRDialect-DAG: %[[Y:.*]] = fir.alloca i32 {name = "{{.*}}Ey"}
+!FIRDialect-DAG: %[[Z:.*]] = fir.alloca i32 {name = "{{.*}}Ez"}
+
+!FIRDialect-DAG: %[[LARGE:.*]] = constant 1 : i32
+!FIRDialect-DAG: %[[DEFAULT:.*]] = constant 1 : i32
+  
+!FIRDialect: omp.parallel private(%[[X]] : !fir.ref<i32>, %[[Y]] : !fir.ref<i32>, %[[Z]] : !fir.ref<i32>, %[[A]] : !fir.ref<i32>, %[[B]] : !fir.ref<i32>) allocate(%[[LARGE]] : i32 -> %[[X]] : !fir.ref<i32>, %[[LARGE]] : i32 -> %[[Y]] : !fir.ref<i32>, %[[LARGE]] : i32 -> %[[Z]] : !fir.ref<i32>, %[[DEFAULT]] : i32 -> %[[A]] : !fir.ref<i32>, %[[DEFAULT]] : i32 -> %[[B]] : !fir.ref<i32>) {
```

Comment:
Can these be split into multiple lines?

Reply:
Yes, I will split it in the next update.